layouts/partials/head.html

```diff
@@ -4,6 +4,8 @@
   <meta http-equiv="content-language" content="{{ . }}" />
   {{ end }}
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="theme-color" content="#{{ .Site.Params.iosLightOverride | default "ffffff" }}" media="(prefers-color-scheme: light)" />
```

Comment:
Make use of the already defined classes/colors for tailwind. Much rather have a different HTML implementation to leverage those classes that this+more configs.

Reply:
@nunocoracao maybe I'm missing something obvious (I haven't used CSS in a while, or modern CSS ever), but when I tried to use CSS colors in these meta tags, they weren't populated quickly enough (or just didn't work) for iOS to change the notch color. I agree that this is a terrible workaround, but it works for now barring a more extensive set of changes.

Reply:
@Capprin there is people using this with different themes and colors. These feature needs to be seamless otherwise it will lead to a breaking change. This should use one of the colors defined in the theme. Maybe not via a meta tag but doable.